runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -743,8 +742,10 @@ private void applyForEachAddress(BiConsumer<Long, ILogData> logDataProcessor) {
                 final long lower = nextRead;
                 final long upper = Math.min(lower + batchReadSize - 1, logTail);
                 nextRead = upper + 1;
+
+                // Don't cache the read results on server for fast loader
                 Map<Long, ILogData> range =
-                        runtime.getAddressSpaceView().fetchAll(ContiguousSet.create(
+                        runtime.getAddressSpaceView().nonCacheFetchAll(ContiguousSet.create(
```

Comment:
please reformat this piece of code, it is pretty inconvenient to read
```
                ContiguousSet<Long> addresses = ContiguousSet.create(
                        Range.closed(lower, upper),
                        DiscreteDomain.longs()
                );
                Map<Long, ILogData> range = runtime.getAddressSpaceView().fetchAll(addresses, true);
```

Reply:
Done.